craft_platforms/charm/_build.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of craft-platforms.
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Charmcraft-specific platforms information."""
+
+import itertools
+from collections.abc import Sequence
+
+from craft_platforms import _architectures, _buildinfo, _distro, _platforms
+
+DEFAULT_ARCHITECTURES = (
+    _architectures.DebianArchitecture.AMD64,
+    _architectures.DebianArchitecture.ARM64,
+    _architectures.DebianArchitecture.PPC64EL,
+    _architectures.DebianArchitecture.RISCV64,
+    _architectures.DebianArchitecture.S390X,
```

Comment:
No armhf here, but I suppose this is intentional.

Reply:
Correct - Juju doesn't run on armhf and has no plans to do so.

Arguably I could drop riscv64 too, but I've chosen not to as of now because it's "up-and-coming" rather than "old hat"